### PR TITLE
 net-im/signal-desktop-bin: new version 1.5.2

### PR DESCRIPTION
**Pull request by robert7k**

### CODE REVIEW

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit gnome2-utils unpacker xdg-utils pax-utils
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Please sort lexically.

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+DESCRIPTION="Signal Desktop"
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Could you elaborate a little bit on what it is?

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	"
+
+QA_PREBUILT="opt/Signal/signal-desktop
+	opt/Signal/libnode.so
+	opt/Signal/libffmpeg.so"
+
+S="${WORKDIR}"
+
+src_install() {
+	doins -r .
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: But you didn't set any directory with `insinto`!

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild (hunk 1 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Done

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild (hunk 2 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Done

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild (hunk 3 above)
Comment (robert7k) [reply net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Done

**Comment by robert7k** [comment]
I adjusted the installation path for the docs.

File: net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild [hunk 4]
```diff
@@ -33,7 +32,12 @@ QA_PREBUILT="opt/Signal/signal-desktop
 S="${WORKDIR}"
 
 src_install() {
-	doins -r .
+	insinto /
+	dodoc usr/share/doc/signal-desktop/*
```
Comment (mgorny) [inline net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild]: Please don't use wildcards when unnecessary. Use `-r usr/share/doc/signal-desktop/.` instead.

**Pull request closed without merging**